askbot/admin.py

```diff
@@ -141,8 +141,8 @@ def is_published(self, obj):
 
 class PostRevisionAdmin(admin.ModelAdmin):
     list_display = ('id', 'post_id', 'thread_name', 'revision', 'revised_at', 'author', 'approved')
-    list_filter = ('approved',)
-    search_fields = ('post__id',)
+    list_filter = ('approved', 'author')
```

Comment:
Doesn't that list all users in the PostRevision admin list sidebar? That is probably not very usable when the site has thousands of users, so might be better to implement a SimpleListFilter for the author username.